src/gui/previewselect.h

```diff
@@ -63,11 +64,17 @@ class PreviewSelect: public QDialog, private Ui::preview
 protected slots:
     void on_previewButton_clicked();
     void on_cancelButton_clicked();
+    void on_preview_finished(int);
 
 private:
     QStandardItemModel *m_previewListModel;
     PreviewListDelegate *m_listDelegate;
     BitTorrent::TorrentHandle *const m_torrent;
+    bool headerState = false;
+    void restoreUi(Preferences *const pref);
```

Comment:
Access Preferences instance in the function, no need to pass it.

Also move it above `m_previewListModel` and add an empty line below.

Reply:
`m_previewListModel` moved, but don't understand this link https://github.com/qbittorrent/qBittorrent/commit/7da96138b46cfec7e75cdee09732b903d765d8ca#r138264558, it only shows to my commit :), can you send the link again thx

Reply:
it's just https://github.com/qbittorrent/qBittorrent/pull/7429#discussion_r138264558

Reply:
Got it